index.html

```diff
@@ -48,8 +50,238 @@ <h1 class="main-name">Erik Stoupignan</h1>
         <div class="prueba">
             <img src="/assets/image_geometry_2.png" alt="Decorative photo" class="background-img2">
         </div>
-
     </main>
+
+    <!-- Proyects section -->
+    <h2>Projects</h2>
+    
+    <div class="project">
+
+            <!-- Section  #1-->
+            <div class="container">
+                <img class="box1" src="/assets/Rectangle21.png" alt="Decoration photo">
+                <div class="box box2">
+                    <p class="subtitle">Project names goes here</p>
```

Comment:
- [ ] Kindly make all the project titles use a header tag `hx` for proper use of HTML semantics.

Reply:
Ok, I'm already added. Thank you for you advise.